lib/cylc/graph_parser.py

```diff
@@ -270,6 +290,18 @@ def parse_graph(self, graph_string):
         # If debugging, print the final result here:
         # self.print_triggers()
 
+    @classmethod
+    def _report_invalid_lines(cls, lines):
```

Comment:
Please add doc string.

Reply:
I thought I had sent this question already, but apparently I didn't.

Is there a preferred pydoc style? reST? Google? Numpy? I think I've seen Google a few times, but this file I don't think matches any of the standard ones?

Reply:
Thanks. I've put in something.